guessinggame.js

```diff
@@ -13,26 +13,6 @@ async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-
-  let min = 1;
-  let max = 100;
-  let guess = Math.floor(Math.random() * (+max - +min) + +min);
-  let yesOrNo = await ask("Is your number " + guess + "? (Y/N)");
-
-  while (yesOrNo == 'N') {
-    let highOrLow = await ask("Is your number higher or lower? (H/L)")
-
-    if (highOrLow == 'H') {
-      min = guess
```

Comment:
After this redefine `guess` and ask if the new guess is the correct number.  Some for the `else if` below.